assets/base.css

```diff
@@ -354,73 +354,73 @@ h5,
   font-family: var(--font-heading-family);
   font-style: var(--font-heading-style);
   font-weight: var(--font-heading-weight);
-  letter-spacing: 0.06rem;
+  letter-spacing: calc(var(--font-heading-scale) * 0.06rem);
   color: rgb(var(--color-foreground));
   line-height: 1.3;
```

Comment:
When I update the size, the filter title doesnt seem to change: https://screenshot.click/28-05-hftxj-r8zyo.mp4

Code:
https://screenshot.click/28-08-9zifa-8am6p.png

Reply:
Updated. The filter drawer heading now scales with `heading_scale` setting

![Screen_Shot_2021-09-28_at_3_03_12_PM](https://user-images.githubusercontent.com/3710548/135149919-9f5633bf-f5f7-4a19-9246-3b797fbaad90.jpg)